README.md

```diff
@@ -102,17 +102,17 @@ The dashboard uses node v20.12.2. To build/run the dashboard on your local machi
 #### install requirements
 We use npm for local package management. To install the requirements:
 ```bash
-  $ nvm use
-  $ npm ci
+  nvm use
+  npm ci
 ```
 
-Use `$ npm install` when package.json is updated.
+Use `npm install` when package.json is updated.
 
```

Comment:
Add a line for `npm install connected-react-router@6.9.3 --legacy-peer-deps` (please see earlier comment)